scheduler/scheduler.go

```diff
@@ -151,25 +188,57 @@ func podFromJob(
 		Value: job.Uuid,
 	})
 	for k, v := range envMap {
-		env = append(env, corev1.EnvVar{Name: k, Value: v})
+		switch k {
+		case "BUILDKITE_PLUGINS": //noop
```

Comment:
You have a // noop comment here, but you are appending the EnvVar in this case?

Reply:
Good call, I think I was wrestling with how this would work and thought at one point that overriding plugins like this was necessary. It's not, we can drop the env var completely.